docs/user_guide/resource_quota_mgmt.md

```diff
@@ -150,3 +150,38 @@ Jobs continue to be submitted to namespaces, based on the `Placementrule` used
 in the configuration. YuniKorn will automatically run the job and all its pods in
 the corresponding queue. For example, if a job is submitted to namespace `development`,
 then you will see the job is running in `root.development` queue.
+
+## Option 3) Custom mapping of workspaces to queues using Kubernetes shim
+
+You can configure the following annotation for a Kubernetes namespace in your cluster: `yunikorn.apache.org/parentqueue`.
+Each pod (allocation) created in the namespace will be passed to the scheduler with the value of that annotation under the `namespace.parentqueue` tag.
+
+### Example: Creating dynamic leaf queues based on namespaces
+
+Cluster administrators can use this behaviour to put allocations in dedicated queues based on the namespace's annotations.
+
+Let's say we have the following queue configuration:
+```yaml
+placementrules:
+ - name: user
+   create: true
+   parent:
+     - name: tag
+       value: namespace
+       create: true
+       parent:
+         - name: tag
+           value: namespace.parentqueue
+queues:
+ - name: root
+   queues:
+     - name: namespaces
+```
+and the `production` namespace is annotated with the following annotation:
+```yaml
+yunikorn.apache.org/parentqueue: root.namespaces
+```
+
+Admins can put each application into different user's queue based on the namespace setting using the settings above.
+
+Let's say user `bob` submitting an application to the `production` namespace. The application will be placed onto `root.namespaces.production.bob`.
```

Comment:
is this correct? I was under the impression the app will be placed to `root.namespaces.production`. Why there is a user queue under production? Can we simplify the example? Change to the path like "root.production.bob".

Reply:
Actually you don't need to annotate the namespace if you want to achieve that.
By changing the queue hierarchy to produce a config which produces the `root.production.bob` as an answer, you don't use the annotation on the namespace at all.

I simplified the example a bit, but we do need that layer - that is what this whole example is about.

Reply:
I mean can we set up the static queues like:

```
placementrules:
 - name: tag
   value: namespace
   create: true
   parent:
     - name: tag
       value: namespace.parentqueue
queues:
 - name: root
   queues:
     - name: production
     - name: development
```

and then we submit an app to a namespace `bob`, then we expect to see the app will be running under `root.production.bob`. Having `namespaces` as the parent queue name is a bit confusing to me.

Reply:
The `bob` namespace is a bit odd, since it's rather a user. I replaced it with `finance` and modified the example. Could you double check @yangwwei ?

Reply:
Sure. Looks good now. Thanks for making these changes. +1